src/screens/Onboarding/WelcomeScreen.tsx

```diff
@@ -53,7 +53,7 @@ const WelcomeScreen = ({
       <Illustration Svg={Onboarding1} />
       <ScrollView style={styles.mainView}>
         <View ref={focusRef} accessibilityRole="header" accessible={true}>
-          <ThemeText type={'body__primary--jumbo--bold'} color={themeColor}>
+          <ThemeText type={'heading__title'} color={themeColor}>
```

Comment:
Jeg ser ikke denne endringen i skissene?

Reply:
jeg hadde glemt å kjøre yarn for å få inn de siste design-endringene. jeg reverter.